src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy

```diff
@@ -60,78 +64,129 @@ import wooga.gradle.releaseNotesGenerator.utils.ReleaseBodyStrategy
-            project.pluginManager.apply(ReleaseNotesGeneratorPlugin)
+            project.pluginManager.with {
+                apply(GrgitPlugin)
+                apply(GithubReleaseNotesPlugin)
+            }
+            configureReleaseNotesTask(project)
+            configureGithubPublishTask(project)
+
+            // TODO: Remove?
+            configureVersionCode(project)
+
+            configureUnityPackageIfPresent(project, extension)
+            configureSetupTaskIfUnityPluginPresent(project)
+            configurePaketConfigurationArtifacts(project)
+        } else {
+            project.rootProject.pluginManager.apply(this.class)
         }
 
-        configureVersionCode(project)
-        configureUnityPackageIfPresent(project, extension)
-        configureSetupTaskIfUnityPluginPresent(project)
-        configurePaketConfigurationArtifacts(project)
+
     }
 
     private static void configureReleaseLifecycle(final Project project) {
         def tasks = project.tasks
         def setup = tasks.maybeCreate(SETUP_TASK)
 
+        // TODO: Add custom tasks previously provided by the nebula release plugin
+        Task finalTask = project.tasks.create(FINAL_TASK)
+        Task rcTask = project.tasks.create(RC_TASK)
+        Task snapshotTask = project.tasks.create(SNAPSHOT_TASK)
+
         //hook up into lifecycle
         def checkTask = tasks.getByName(LifecycleBasePlugin.CHECK_TASK_NAME)
         def assembleTask = tasks.getByName(LifecycleBasePlugin.ASSEMBLE_TASK_NAME)
         def buildTask = tasks.getByName(LifecycleBasePlugin.BUILD_TASK_NAME)
-        def releaseTask = tasks.getByName('release')
-        def postReleaseTask = tasks.getByName(nebula.plugin.release.ReleasePlugin.POST_RELEASE_TASK_NAME)
+
         def publishTask = tasks.getByName(PublishingPlugin.PUBLISH_LIFECYCLE_TASK_NAME)
         def testTask = tasks.maybeCreate(TEST_TASK)
-        def unityPack = tasks.getByName(UNTIY_PACK_TASK)
+
+        [finalTask, rcTask, snapshotTask].each {
+            it.dependsOn publishTask
+        }
 
         testTask.dependsOn setup
         checkTask.dependsOn(testTask)
         buildTask.dependsOn setup
-        assembleTask.dependsOn unityPack
-        releaseTask.dependsOn assembleTask
-        postReleaseTask.dependsOn publishTask
-        publishTask.mustRunAfter releaseTask
+        publishTask.dependsOn assembleTask
+    }
+
+    private static void configureReleaseNotesTask(Project project) {
+
+        // Description
+        addReleaseNotesTask(project, RELEASE_NOTES_DESCRIPTION_TASK, new ReleaseNotesDescriptionStrategy(), { t ->
+            t.output.set(new File("${project.buildDir}/outputs/release-notes.md"))
+        })
+
+        // Page
+        // TODO: Implement; should path be at the root of repo?
+//        addReleaseNotesTask(project, RELEASE_NOTES_MD_TASK, new ReleaseNotesPageStrategy(), { t ->
+//            t.output.set(new File("${project.buildDir}/RELEASE_NOTES.md"))
+//        })
```

Comment:
The output of project specific Items should always be the `build` dir of the respected project.

Reply:
Very well.